client/src/partials/templates/bhLocationSelect.tmpl.html

```diff
@@ -3,11 +3,11 @@
   a child form is added for validation purposes, and to make the HTML easier
   to read.
 -->
-<fieldset ng-disabled="$ctrl.disable" ng-form="LocationSelectForm" data-bh-location-select>
+<fieldset ng-disabled="$ctrl.disable"  ng-form="{{$ctrl.name}}" ng-model="somevaluetwo" data-bh-location-select>
 
   <!-- allows a user to select a country -->
   <div class="form-group"
-    ng-class="{ 'has-error' : $ctrl.validationTrigger && LocationSelectForm.country.$invalid }"
+    ng-class="{ 'has-error' : $ctrl.validationTrigger && this[$ctrl.name].country.$invalid }"
```

Comment:
`$timeout` works in this case. I understand that knowing that `this` is the `$scope` injected into the controller can be an abstract concept to understand. 

The benefit of the `$timeout` alias is that you only have to explain it in comments once and then use the alias from then on. 

If requiring an additional dependency is an issue I'm sure something could be done with `$scope.$watch`. 

I'm not sure any of these are good long term solutions but they're all I can think of off the top of my head.